examples/2d/frustum_sprites.rs

```diff
@@ -0,0 +1,83 @@
+use bevy::{
+    diagnostic::{FrameTimeDiagnosticsPlugin, LogDiagnosticsPlugin},
+    prelude::*,
+};
+
+const CAMERA_SPEED: f32 = 1000.0;
+
+pub struct CameraTimer(Timer);
+pub struct Direction(Vec3);
+
+fn main() {
+    App::build()
+        .insert_resource(ReportExecutionOrderAmbiguities)
+        .add_plugin(LogDiagnosticsPlugin::default())
+        .add_plugin(FrameTimeDiagnosticsPlugin::default())
+        .insert_resource(bevy::log::LogSettings {
+            level: bevy::log::Level::DEBUG,
+            ..Default::default()
+        })
+        .add_plugins(DefaultPlugins)
+        .add_startup_system(setup.system())
+        .add_system(move_camera.system())
+        .run()
+}
+
+fn setup(
+    commands: &mut Commands,
+    assets: Res<AssetServer>,
+    mut materials: ResMut<Assets<ColorMaterial>>,
+) {
+    let tile_size = Vec2::new(64.0, 64.0);
+    let map_size = Vec2::new(320.0, 320.0);
```

Comment:
These can both use `Vec2::splat`. Not sure if that is clearer, but it might be better than repeating the constant.